### DISCUSSION TITLE
Style Guide: Typesetting Guidelines?

### DISCUSSION THREAD

**Discussion by cg2wilson** [TeamBasedInquiryLearning/library#63] [General]
This discussion is here as a one-stop shop to collect ideas/thoughts related to the Style Guide and formatting things like limits (a la https://github.com/TeamBasedInquiryLearning/calculus/pull/140) or #46

**Comment by StevenClontz** [comment]
Relevant wiki page: https://github.com/TeamBasedInquiryLearning/library/wiki/Style-Guide (and I've linked back here)

**Comment by cg2wilson** [comment]
One thing that I'm finding is that there's a slight inconsistency in display use; some of this may be an artifact of a Find/Replace for the limit issue, but still.

Here are some examples of what I mean:
-[Activity 1.3.13](https://teambasedinquirylearning.github.io/calculus/2024/LT3.html#activity-limits-rationals2)  
-[Note 1.3.20](https://teambasedinquirylearning.github.io/calculus/2024/LT3.html#inst-velocity) 

The public version at the time of writing still has some non-deployed typos, but the display issue should still be visible.

My opinion is that fractions should be \dfrac rather than \frac everywhere except possibly when doing a complex fraction like 1/(x+1/x). Does anyone have opinions one way or the other?

**Reply by StevenClontz** [reply]
Never used `\dfrac`! Can you share some screenshots to compare?

**Reply by cg2wilson** [reply]
My understanding/use is that `\dfrac` is a built-in shortcut for `\displaystyle\frac`. Here are screenshots showing the difference in DF1:
![Screenshot 2024-07-23 125919](https://github.com/user-attachments/assets/e20cdbd6-7735-438f-ba0b-9cc9f74cf3ae)
for the `\frac` version, and
![Screenshot 2024-07-23 130321](https://github.com/user-attachments/assets/8a0f67ef-3bbf-42b0-8b47-a08d4fea2189)
for the `\dfrac` version.

I prefer using `\dfrac`s because they render larger and more readable. That larger render does sometimes make the fraction physically close to the preceding or following paragraph line. Here's a comparison of Activity 1.3.13 with and without the in-line `\dfrac`s.
![Screenshot 2024-07-23 131339](https://github.com/user-attachments/assets/f5c10adb-6d2d-4acb-ae7e-c0fdbb4caf4a)
With the `\dfrac`s
![Screenshot 2024-07-23 131458](https://github.com/user-attachments/assets/91005c64-d79e-4fbe-80d8-21833a446f79)
Without them

**Reply by StevenClontz** [reply]
I don't have a strong opinion here; @siwelwerd?

**Reply by siwelwerd** [reply]
I'm open to this change, but want to check what it looks like in the PDF version as well.  @cg2wilson Any chance you could show us a side by side image from the PDF?  Assuming that looks similar, I'm happy to move forward with this.

**Reply by cg2wilson** [reply]
Here are screenshots from the pdf:
![obs2112-no](https://github.com/user-attachments/assets/8f46e70d-02d1-478b-83bd-0aec7f48b528)
Observation 2.1.12 without the `\dfrac`, with surrounding text
![obs2112-yes](https://github.com/user-attachments/assets/bd66e6b9-009a-4532-ad25-8c46706ed950)
This time with the `\dfrac`
![act1313-no](https://github.com/user-attachments/assets/78179340-812d-48ff-9393-c459aacee089)
Activity 1.3.13 without `\dfrac`s, in context
![act1313-yes](https://github.com/user-attachments/assets/a774071e-cb16-4490-91db-958bfcdcd1d7)
Activity 1.3.13 with `\dfrac`s

**Reply by siwelwerd** [reply]
I'm happy to have that implemented as an "official" style guide, and then we'll need to go through and update them all.  It probably affects mostly calc and precalc.

**Reply by cg2wilson** [reply]
i'll make that an issue and assign it to myself

**Reply by jbunn3** [reply]
For what it's worth at this point, I almost always use \dfrac unless I have a compelling reason to use \frac.

**Comment by AbbyANoble** [comment]
Style question/discussion: When writing a multiple choice question, how many columns do we want for the answer choices?

This is coming from me putting questions on the screen today and not being able to see a the original question on an activity with multiple parts without zooming out a lot. (Which is my own fault because it's a section I wrote and I tend to put answers in one column by default almost. See [Section 2.5: Combining and Composing Functions (FN5)](https://library.tbil.org/2024/precalculus/FN5.html). 

My thoughts to help with viewing--- 

- If the answers are very short and look okay and fit side by side, like a number, then the number of columns should be the number of answers. (All on one line, basically. **Not** like this: [Activity 2.5.9](https://library.tbil.org/2024/precalculus/FN5.html#activity-fn5-composition-3). This is where I scrolled a lot in class today. Again, my own fault as I wrote it. Haha.)
- If the answers are fairly long, just one column. (Like this: [Activity 2.5.4(b)](https://library.tbil.org/2024/precalculus/FN5.html#activity-fn5-composition-1-3)  Multiple columns would "squeeze" this too much in my opinion.)
- If the answers are in between, then ??? Maybe 2 columns?

I guess the question is how do we define "fairly long"?

In calculus, I see that some of the section authors defaulted to two columns for most everything.  Like in [Section 2.1: Derivatives Graphically and Numerically (DF1)](https://library.tbil.org/2024/calculus/DF1.html).
 I think this is good sometimes and saves some screen space, like shown in [Activity 2.1.20(b)](https://library.tbil.org/2024/calculus/DF1.html#tangent-line-estimate-3)   (screenshot below) 

![image](https://github.com/user-attachments/assets/a47a138e-d967-48a7-a5ff-4f740c1f9198)

But when the answers are longer it looks a little weird to me. Like if the answers all cause multiple lines in two columns, maybe we should just use one column. Like in [Activity 2.1.14](https://library.tbil.org/2024/calculus/DF1.html#estimating-der2) 
 (screenshot below)

![image](https://github.com/user-attachments/assets/6235a659-b409-48c7-860f-8f220d3236ea)

And then in the slides, this same one does not look so great as LaTeX forces the justified column widths and equal column heights.

![image](https://github.com/user-attachments/assets/ad798a31-5c63-44e6-b7a7-e8adae2c5f5e)


What does everyone else think?

**Reply by StevenClontz** [reply]
I think using multiple columns only when answers take up the same amount of space (roughly) makes sense.

**Comment by siwelwerd** [comment]
How do we feel about $\sin \theta$ versus $\sin(\theta)$?  Do our feelings change if it is something like $\sin 2\theta$ or $\sin ^2 2\theta$?

**Reply by AbbyANoble** [reply]
I'm a $\sin\theta$ fan over  $\sin(\theta)$. I would switch to parentheses only for things like $\sin(\theta-\pi)$ or something where they are necessary for grouping things together, but not for the others you mentioned. Extra parentheses feel cluttered to me.

**Reply by siwelwerd** [reply]
We also discussed this in our Precalculus authoring meeting today, and nobody felt super strongly about this.  So at least for now we are proceeding with mixing it up as seems appropriate in the given context.
